jsapp/js/account/security/email/emailSection.component.tsx

```diff
@@ -55,15 +58,20 @@ export default function EmailSection() {
   }, []);
 
   function setNewUserEmail(newEmail: string) {
-    setUserEmail(newEmail).then(() => {
-      getUserEmails().then((data) => {
-        setEmail({
-          ...email,
-          emails: data.results,
-          newEmail: '',
+    setUserEmail(newEmail).then(
```

Comment:
Unless my eyes are missing something, I don't think you actually touched the code here. Looks like an automatic lint change. We want to avoid noise like this in PRs so the code changes are easier to read. Can you revert it?